docs/outline.toml

```
@@ -372,6 +372,11 @@ title = "Jira Tasks"
 module = "prefect.tasks.jira"
 classes = ["JiraTask", "JiraServiceDeskTask"]
 
+[pages.tasks.monday]
+title = "Monday Tasks"
+module = "prefect.contrib.tasks.monday"
```

Incorrect import path, I think this is causing your tests to fail:
```suggestion
module = "prefect.tasks.monday"
```